src/components/button/Button.tsx

```diff
@@ -5,7 +5,7 @@ interface ButtonProps {
   /**
    * Is this the principal call to action on the page?
    */
-  primary?: boolean;
+  isPrimary?: boolean;
```

Comment:
Out of scope so non-blocking, but I'd prefer if we don't use booleans for things that just happens to have only two options at the moment.

```suggestion
  type?: 'primary' | 'secondary';
```